presentation/src/main/java/com/architect/coders/mu8/characters/detail/CharactersDetailViewModel.kt

```diff
@@ -4,11 +4,11 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 
-class CharactersDetailViewModel() : ViewModel() {
+class CharactersDetailViewModel : ViewModel() {
 
     private val _model = MutableLiveData<CharactersDetailUiModel>()
     val model: LiveData<CharactersDetailUiModel>
         get() {
             return _model
         }
-}
\ No newline at end of file
+}
```

Comment:
What change was done here?

Reply:
sorry i have detekt automatic and i don't realize the change. I'm going to put it back the way it was.